packages/server/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
These are the presets needed to compile the manifest correctly. Very odd indeed, and something that we'll have to work out since it doesn't apply to the actual _server_ code, but the manifest code that the server compiles with parcel.